### ISSUE TITLE
Link purpose (WCAG 2.4.4) 

### ISSUE THREAD

**Issue by VivienDZB** [slidewiki/slidewiki-platform#1238]
**Request:** The intent of this criterion is to help users understand the purpose of each link so they can decide whether they want to follow the link. Whenever possible, provide link text that identifies the purpose of the link without needing additional context.

**Issues:**
1. https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3
- The page contains links that do not contain any meaningful link text. The link text should provide information about the purpose of the link.

**Hint:** 
#1159
Despite the German language selection, there are many English terms on the tested pages, both visible content and alternative texts (which also effects link texts):  
- https://slidewiki.org/ 
- https://slidewiki.org/presentation/81127-1/overview-of-accessibility/811271/437649-3/?language=en-/slide-719796-3 
- https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/edit 
- https://slidewiki.org/deck/127411/_/slide/836159-2/836159-2:1/markdownEdit 
- https://slidewiki.org/search?keywords=accessibility&sort=score 
- https://slidewiki.org/user/vivien.zschammer/recommendations 
- https://slidewiki.org/addDeck 
- https://slidewiki.org/contactus 
- https://slidewiki.org/deck/127411/test/deck/127411 
- https://slidewiki.org/accessibility 
- https://slidewiki.org/user/vivien.zschammer/settings/profile
- https://slidewiki.org/user/vivien.zschammer/stats

**Comment by rezatavakoli** [comment]
When the link (in the request description) is clicked, an empty presentation is shown and it's not clear that in which links, the purpose should be added. Moreover, the menu button on the left-bottom has been removed in the latest version.

Also, I really appreciate if you explain more about the "Hint" part.

**Comment by VivienDZB** [comment]
Oh, I think the link is no longer the right one. It should be this one: 
https://slidewiki.org/presentation/81127-1/overview-of-accessibility/81127-1/?language=en#/slide-437649-3

According to the WCAG, users should be able to unterstand the purpose of every link. This means that the link text should identify the purpose of the link without needing additional context. And it also means that the link text is in the same language as the rest of the page. Since the German SlideWiki page has not yet been uniformly translated, this also means that link texts are sometimes English or German and may therefore not be understood by some users. This is why I added #1159 as a hint here.

**Comment by aoelen** [comment]
If I understand you correctly (apart from the non-translated links) you are talking about the links within a presentation, correct? Because for this, users themselves are responsible to add the appropriate description there. They can do so by providing a display text, see: 

![image](https://user-images.githubusercontent.com/1567948/99083472-e4107180-25c5-11eb-991e-037899281f57.png)

So I think this issue can be closed, if not, feel free to reopen.
